sorc/Externals.cfg

```diff
@@ -2,8 +2,8 @@
 protocol = git
 repo_url = https://github.com/ufs-community/UFS_UTILS
```

Comment:
I am going to create a tag for you. Don't use the release branch.

Reply:
Sounds good, I can update that.

Reply:
Please use this tag: https://github.com/ufs-community/UFS_UTILS/releases/tag/ufs_utils_1_12_2

I am going to delete the release branch. If you need further updates, I can create a new release branch from the tag.

Reply:
Thanks @GeorgeGayno-NOAA for being willing to create a fresh release branch if it becomes needed.